about.css

```diff
@@ -0,0 +1,510 @@
+  flex-direction: column;
+  align-items: center;
+  gap: 30px;
+  padding: 50px 5vw;
+  background: #272a31;
+  color: white;
+}
+
+.section-4 .h2-red-line {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 10px;
+}
+
+.section-4 .red-line {
+  height: 1px;
+  width: 50px;
+  background-color: #ec5242;
+}
+
+.section-4 .image-container {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.section-4 .image {
+  height: 50px;
+  width: 150px;
+}
+
+/* Footer Section */
+
+.footer {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  gap: 30px;
+  padding: 50px 5vw;
+  background: #fff;
+  color: #9b9b9b;
+}
+
+.footer .logo {
+  height: 30px;
+  width: 160px;
+}
+
+.footer .text-inner-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 5px;
+  white-space: nowrap;
+  font-size: 13px;
+  color: #3b3e45;
+}
+
+.footer .licence {
+  display: none;
+}
+
+/* Desktop */
+@media only screen and (min-width: 768px) {
+  .header-section1-container {
+    background-image: none;
+    padding: 0;
+  }
+
+  /* Header */
+
+  .header .hamburger {
+    display: none;
+  }
+
+  .header .desktop-nav {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .header .logo {
+    height: 40px;
+    width: 105px;
+  }
+
+  .header .desktop-nav .black-row {
+    display: flex;
+    justify-content: end;
+    align-items: center;
+    gap: 20px;
+    background: #272a31;
+    padding: 10px 15vw;
+    color: white;
+    font-size: 10px;
+  }
+
+  .header .desktop-nav .white-row {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background: white;
+    padding: 15px 15vw;
+  }
+
+  .header .desktop-nav .white-row .list {
+    display: flex;
+    gap: 20px;
+  }
+
+  .header .desktop-nav .white-row .a {
+    color: #9d9d9d;
+    font-size: 15px;
+  }
+
+  .header .desktop-nav .white-row .a:hover {
+    color: #ec5242;
+  }
+
+  .header .desktop-nav .white-row .a-cc {
+    border: solid 4px red;
+    padding: 10px;
+    color: #ec5242;
+    white-space: nowrap;
+  }
+
+  /* Section 1 */
+
+  .section-1 {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 20px;
+    margin: 0;
+    padding: 150px 20vw 150px 20vw;
+    text-align: center;
+    background-image: url('images/home-bg-desktop.png');
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  .section-1 .hello {
+    display: none;
+  }
+
+  .section-1 .h1 {
+    color: #ec5242;
+    font-size: 50px;
+    line-height: 50px;
+    margin-bottom: 0;
+    max-width: 460px;
+  }
+
+  .section-1 .red-line {
+    height: 1px;
+    width: 25px;
+    background: #ec5242;
+  }
+
+  .section-1 .p {
+    background: white;
+    border: 1px solid #e3e2e2;
+    padding: 20px;
+    line-height: 25px;
+    font-size: 14px;
+    color: #7a7a7a;
+    max-width: 715px;
+  }
```

Comment:
- [ ] As suggested, please make sure to use the given colors  for `.section-1 .p` & `.contact` The basic color swatch is 👉 `#272A31, #EC5242, #D3D3D3`


Design before | Design now 
---| ---|
![image](https://github.com/suthish777/capstone__01/assets/106119105/314dc6db-a9c5-42eb-9670-a47144a2bd25) | ![image](https://github.com/suthish777/capstone__01/assets/106119105/d43d2c5a-46be-40ee-a43d-12ba0497dd1e)